x-osu.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mime-info xmlns='http://www.freedesktop.org/standards/shared-mime-info'>
+    <mime-type type="application/x-osu-beatmap">
+        <comment>osu! beatmap</comment>
+        <comment xml:lang="hk">osu! 譜面</comment>
+        <comment xml:lang="zh">osu! 谱面</comment>
+        <comment xml:lang="ru">Карта osu!</comment>
+        <icon name="sh.ppy.osu.x-osu-beatmap"/>
+        <glob pattern="*.osu"/>
+    </mime-type>
+    <mime-type type="application/x-osu-beatmap-archive">
+        <sub-class-of type="application/zip"/>
+        <sub-class-of type="application/x-rar"/>
```

Comment:
Same thing, I don't think archive types can be `application/x-rar`, only `application/zip`. Correct me if I'm wrong though.

Reply:
![IMG_20211114_133910.jpg](https://user-images.githubusercontent.com/48867887/141677521-dfd91c15-10bb-49ad-ad7e-180070d1db8d.jpg)

Sorry, I got it a little confused, not having noticed the point about the mandatory zip format below in the article